File: Swift-C++-IR-Comparison/test/controlFlow/Swift/NestedWhileLoopLabledBreak.swift
@main
struct MyMain {

    static func main() {
        var counter = 0
        var max = 12
    outsideLoop: while counter < max {
        counter += 1
        switch(counter) {
        case 1...4:
            continue outsideLoop
        default:
            break outsideLoop
        }
        }
        print(counter)
    }
}

File: Swift-C++-Testsuite/AutoTranspiledTests/try-0/NestedWhileLoopLabledBreak.cpp

 #include <iostream>
 
int main() {
    int counter = 0;
    int max = 12;

    while (counter < max)
    {
        counter += 1;

        switch (counter)
        {
        case 1 ... 4:
            continue;
        default:
            break;
        }
    }

    std::cout << counter << std::endl;
    
    return 0;
}